modules/apps/headless/headless-batch-engine/headless-batch-engine-test/src/testFunctional/tests/HeadlessBatchEngineExportOrganization.testcase

```diff
@@ -0,0 +1,83 @@
+@component-name = "portal-headless"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Headless Batch Engine API";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		for (var externalReferenceCodeList : list "sub_organizationErc,organizationErc") {
+			OrganizationAPI.deleteOrganizationByExternalReferenceCode(externalReferenceCode = ${externalReferenceCodeList});
+		}
+
+		var testLiferayVirtualInstance = PropsUtil.get("test.liferay.virtual.instance");
+
+		if (${testLiferayVirtualInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@priority = 4
+	test CanExportAllOrganization {
+		property portal.acceptance = "true";
+
+		task ("Given a new organization") {
+			JSONOrganization.addOrganization(
+				externalReferenceCode = "organizationErc",
+				organizationName = "testOrganization");
+		}
+
+		task ("And Given a sub_tesOrganization in the created testOrganization") {
+			JSONOrganization.addOrganization(
+				externalReferenceCode = "sub_organizationErc",
+				organizationName = "sub_testOrganization",
+				parentOrganizationName = "testOrganization");
+		}
+
+		task ("And Given using POST export-task with className com.liferay.headless.admin.user.dto.v1_0.Organization and parameter flatten=true") {
+			var response = ExportTask.createExportTask(
+				className = "com.liferay.headless.admin.user.dto.v1_0.Organization",
+				contentType = "JSON");
+
+			var externalReferenceCode = JSONPathUtil.getErcValue(response = ${response});
+		}
+
+		task ("And Given using GET export-task/by-external-reference-code/${externalReferenceCode}/content with externalReferenceCode from POST") {
+			APIExplorer.navigateToOpenAPI(
+				api = "headless-batch-engine",
+				version = "v1.0");
+
+			APIExplorer.executeAPIMethod(
+				method = "getExportTaskByExternalReferenceCodeContent",
+				parameter = "externalReferenceCode",
+				service = "ExportTask",
+				value = ${externalReferenceCode});
+		}
+
+		task ("When click execute and download file") {
```

Comment:
`When click execute and download file` we already execute in the upper step and only need to download a file. Please help update the description